### DISCUSSION TITLE
사진을 불러올 때 419에러가 납니다

### DISCUSSION THREAD

**Discussion by cozups** [morethanmin/morethan-log#173] [Q&A]
안녕하세요! 블로그 잘 사용하고 있습니다.
게시물에 사진이 있을 때 로딩이 안돼서 엑박이 뜨네요...
네트워크 탭을 켜보니 419에러가 뜹니다.

![image](https://user-images.githubusercontent.com/58796245/234529265-735ee4c2-5e33-4fab-9fe0-8da7c5512281.png)
![image](https://user-images.githubusercontent.com/58796245/234529369-6bb7e17a-e93d-4f22-ac73-bf7bae27410f.png)

무슨 원인일까요...?

**Comment by morethanmin** [comment]
해당 게시글 링크 공유해주시면 한번 확인해볼게요 :)
